### PR TITLE
Select style

### PR DESCRIPTION
**Pull request by SaraIsabelSantacruz**
## Summary
- update tech select style

## Screenshot
selectStyle

## Trello Card
https://trello.com/c/wsR4SmYs

### CODE REVIEW

File: app/src/app/components/Sidebar/index.tsx [hunk 1]
```diff
@@ -53,16 +74,41 @@ function Sidebar() {
...
+                onClick={toggleSelectMenu}
+                ref={select}
+              >
+                <span className={styles.optionSelected}>{selectedTech}</span>
+              </div>
+              <ul className={cn(styles.menuSelect, { [`${styles.menuSelectOpen}`]: toogleOpen })}>
+                <li key={ALL_TECHS} className={styles.itemList} onClick={() => handleTechChange(ALL_TECHS)}>
+                  <img
+                    src={checkIcon}
+                    alt="Icon check"
```
Comment (mcallegari10) [inline app/src/app/components/Sidebar/index.tsx:92]: No need for `alt` in icons, you can set it to a blank value `alt=" "`

File: app/src/app/components/Sidebar/index.tsx [hunk 2]
```diff
@@ -17,12 +18,31 @@ import styles from './styles.module.scss';
 function Sidebar() {
   const categoryType = useLocation().pathname.split('/')[1];
   const [sidebarIsOpen, setsidebarIsOpen] = useState(false);
+  const [toogleOpen, settoogleOpen] = useState(false);
```
Comment (mcallegari10) [inline app/src/app/components/Sidebar/index.tsx:21]: Typo 🙈 
```suggestion
  const [toggleOpen, setToggleOpen] = useState(false);
```

File: app/src/app/components/Sidebar/index.tsx [hunk 3]
```diff
@@ -53,16 +74,41 @@ function Sidebar() {
...
         {isUserLoggedIn ? (
           <div className={`column ${styles.contentLinks} start`}>
-            <div className="row m-bottom-3">
+            <div className={`column m-bottom-3 ${styles.selectContainer}`}>
               <span className={styles.techTitle}>Tech:</span>
-              <select className={styles.techSelect} value={selectedTech} onChange={handleTechChange}>
-                <option value="all">Todas</option>
+              <div
+                className={cn(styles.boxTechSelected, {
+                  [`${styles.isOpen}`]: toogleOpen
```
Comment (mcallegari10) [inline app/src/app/components/Sidebar/index.tsx:81]: I think you don't need the interpolation 🤔 
```suggestion
                  [styles.isOpen]: toogleOpen
```
Does that work?

File: app/src/app/components/Sidebar/index.tsx [hunk 4]
```diff
@@ -53,16 +74,41 @@ function Sidebar() {
...
+                })}
+                onClick={toggleSelectMenu}
+                ref={select}
+              >
+                <span className={styles.optionSelected}>{selectedTech}</span>
+              </div>
+              <ul className={cn(styles.menuSelect, { [`${styles.menuSelectOpen}`]: toogleOpen })}>
+                <li key={ALL_TECHS} className={styles.itemList} onClick={() => handleTechChange(ALL_TECHS)}>
```
Comment (BJordi) [inline app/src/app/components/Sidebar/index.tsx:89]: Hmmm :thinking: I don't like this thing of having boilerplate just to add one option. I think you can do something like send a boolean and add that "all values" props internally. Or maybe send an object with the label of that "all values" option. Something like this:
```js
options = hasAllValues ? [{ label: allValuesLabel, }, ...tech ] : tech

{options.map(option => { ... })
```

File: app/src/app/components/Sidebar/index.tsx [hunk 5]
```diff
@@ -17,24 +18,46 @@ import styles from './styles.module.scss';
 function Sidebar() {
   const categoryType = useLocation().pathname.split('/')[1];
   const [sidebarIsOpen, setsidebarIsOpen] = useState(false);
+  const [toogleOpen, setToogleOpen] = useState(false);
```
Comment (mcallegari10) [inline app/src/app/components/Sidebar/index.tsx:21]: Remember the typo! 😄

**Pull request merged**